### PR TITLE
New version: DuckStudio.FufuDevTools version 2024.09.08.2000

### PR DESCRIPTION
**Pull request by fjwxzde**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/173678)

### CODE REVIEW

**Comment by fjwxzde** [comment]
@microsoft-github-policy-service agree company="DuckStudio"

**Comment by fjwxzde** [comment]
> The package manager bot determined there was an issue with installing the application correctly. Please check the application installs correctly. Once repaired, please push an update to your pull request.

I am not sure why this error occurred, but I was able to install and use it successfully during my testing.

Terminal logs:
- [install.log](https://github.com/user-attachments/files/17008791/install.log)
- [install-Administrator.log](https://github.com/user-attachments/files/17008792/install-Administrator.log)

Winget logs:

> I don't know which log is related to the installation, I only executed four commands and got these logs.

- [WinGet-DuckStudio.FufuDevTools.2024.09.08.2000-2024-09-16-13-00-46.068.log](https://github.com/user-attachments/files/17008796/WinGet-DuckStudio.FufuDevTools.2024.09.08.2000-2024-09-16-13-00-46.068.log)
- [WinGet-2024-09-16-12-57-46.837.log](https://github.com/user-attachments/files/17008809/WinGet-2024-09-16-12-57-46.837.log)
- [WinGet-2024-09-16-12-59-56.495.log](https://github.com/user-attachments/files/17008816/WinGet-2024-09-16-12-59-56.495.log)
- [WinGet-2024-09-16-13-00-20.074.log](https://github.com/user-attachments/files/17008839/WinGet-2024-09-16-13-00-20.074.log)
- [WinGetCOM-2024-09-16-13-00-58.076.log](https://github.com/user-attachments/files/17008843/WinGetCOM-2024-09-16-13-00-58.076.log)

Usage verification:

```powershell
> ffdev
Usage: ffdev <工具> <命令> [<参数>...]
> ffdev --version
✓ Version 2024.09.08.2000
> ffdev -v
✓ Version 2024.09.08.2000
> ffdev 账号切换 --edit
> # Successful
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

**Comment by stephengillie** [comment]
Hi @fjwxzde,

This manifest removes the DisplayVersion from the AppsAndFeaturesEntries, which is present in previous manifest versions. This entry should be added to this version, to maintain version matching, and prevent the 'upgrade always available' situation with this package.

(Automated response - build 895.)

**Review by Luna-Grace — APPROVED** [review]

**Comment by Luna-Grace** [comment]
@DuckDuckStudio

**Review by DuckDuckStudio — APPROVED** [review]
~~Looks good.~~

File: manifests/d/DuckStudio/FufuDevTools/2024.09.08.2000/DuckStudio.FufuDevTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+InstallerSwitches:
+  Silent: /verysilent /suppressmsgboxes
+  SilentWithProgress: /verysilent
+UpgradeBehavior: install
+Commands:
+- ffdev
+Protocols:
+- https
+AppsAndFeaturesEntries:
+- DisplayVersion: 2024.08.01.0000
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/FufuDevTools/2024.09.08.2000/DuckStudio.FufuDevTools.installer.yaml:21]:
```suggestion
- DisplayVersion: 2024.09.08.2000
```

I didn't notice this just now.

**Review by DuckDuckStudio — CHANGES_REQUESTED** [review]

**Review by Luna-Grace — APPROVED** [review]

**Review by DuckDuckStudio — APPROVED** [review]

**Comment by fjwxzde** [comment]
Hi @stephengillie ,
There seems to be a problem with the pipeline.
The error label was removed, but the pipeline was not restarted.

**Comment by fjwxzde** [comment]
> Azure Pipelines could not run because the pipeline triggers exclude this branch/path.

Hi @stephengillie ,
There seems to be a problem with the pipeline.
The error label was removed, but the pipeline was not restarted.

**Comment by fjwxzde** [comment]
> Azure Pipelines could not run because the pipeline triggers exclude this branch/path.

Hi stephengillie ,
~~There seems to be a problem with the pipeline.~~
~~The error label was removed, but the pipeline was not restarted.~~

image

This was not the case when I sended this comment.
But I kone it now.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

**Review by stephengillie — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
@stephengillie 
image

**Pull request merged**